azure-pipelines.yml

```diff
@@ -2,32 +2,55 @@
 # Create and test a Python package on multiple Python versions.
 # Add steps that analyze code, save the dist with the build record, publish to a PyPI-compatible index, and more:
 # https://docs.microsoft.com/azure/devops/pipelines/languages/python
-
+pr: none
 trigger:
-- master
-
-pool:
-  vmImage: 'ubuntu-latest'
-strategy:
-  matrix:
-    Python36:
-      python.version: '3.6'
-    Python37:
-      python.version: '3.7'
-    Python38:
-      python.version: '3.8'
-steps:
-- task: UsePythonVersion@0
-  inputs:
-    versionSpec: '$(python.version)'
-  displayName: 'Use Python $(python.version)'
+  branches:
+    include:
+    - '*'
 
-- script: |
-    python -m pip install --upgrade pip
-    pip install -r requirements.txt
-  displayName: 'Install dependencies'
+stages:
+- stage: static_checks
+  displayName: Static Checks
+  dependsOn: []
+  jobs:
+    - job:
+      displayName: "Check Git Tags"
```

Comment:
Do you know what this static check section is doing? I suspect it isn't required anymore. If (later) we wanted to add static checks like a linter or mypy they could go here though.

Reply:
This is for if this build is triggered by a tag, it will add a tag 'Automated' to the current build. Used to 
trigger the release pipeline. I copied from blocklib azure yaml file. Currently I didn't include mypy check since I know it will fail the tests. It will be added and fixed in a separate issue